lms/djangoapps/discussion/static/discussion/js/spec/views/discussion_user_profile_view_spec.js

```diff
@@ -1,278 +1,45 @@
-define(
-    [
+/* globals Discussion, DiscussionCourseSettings */
+(function(define) {
```

Comment:
I don't think you need to do this. Our specs use RequireJS without a namespace, so the way this code was before was correct.